src/course-unit/CourseUnit.jsx

```diff
@@ -98,8 +98,9 @@ const CourseUnit = ({ courseId }) => {
             </Layout.Element>
             <Layout.Element>
               <Stack gap={3}>
-                <Sidebar />
-                <Sidebar isDisplayUnitLocation />
+                <Sidebar variant="publish" />
+                <Sidebar variant="tags" />
+                <Sidebar variant="location" />
```

Comment:
I think these should be different components. As this is coming from upstream, what do you think about separating at least the `TagsSideBar`?

Reply:
That seems weird to me. Why not just make `<Sidebar>` into a wrapper component, so we can use

```
<Sidebar><PublishControls /></Sidebar>
<Sidebar><TagControls /></Sidebar>
<Sidebar><LocationInfo /></Sidebar>
```
or something like that? Of course you need to discuss with the other authors of the upstream PR. And maybe do that later as a follow-on. But that feels a lot cleaner to me.